### ISSUE TITLE
About start index

### ISSUE THREAD

**Issue by JYJ-YeongJoon-Jang** [ycmin95/VAC_CSLR#5]
Hi, thank you for your work.

I'm not sure whether the start index is False or True for KD loss calculation.

Could you let me know about it?

**Comment by ycmin95** [comment]
@JYJ-YeongJoon-Jang 
We adopt the KD loss to distill the logits learnt by the BiLSTM layer to the convolution layer. Due to the spiky activations of CTC, we don't know whether blank labelled logits contain useful information, so we try distill with blank logits and without blank logits. Experimental results show that without blank logits can achieve better performance and we adopt this setting [in this work](https://github.com/ycmin95/VAC_CSLR/blob/main/slr_network.py#L104). 
There are some assumptions about this result: blank logits contain less information, or the unbanlanced ratio of blank and non-blank logits.

**Comment by JYJ-YeongJoon-Jang** [comment]
Thank you for kind and fast reply :)

I'll cite this paper !

best regards
yeongjun jang



-----Original Message-----
